src/target/lpc40xx.c

```diff
@@ -0,0 +1,273 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2012 Gareth McMullin <gareth@blacksphere.co.nz>
```

Comment:
You may wish to update this to credit yourself and have the correct copyright year